content/preferences-settings/shortcuts.md

```diff
@@ -5,87 +5,126 @@ weight: 120
 
-For external devices you do not need to start with a _simple_ shortcut -- you can directly assign a control knob or joystick to an action -- though this will significantly reduce the flexibility of such devices.
+Hence, the most complex shortcut can consist of 8 key/button presses:
+* Three key presses of the same `key`, followed by
+* Three presses of the three mouse buttons, followed by
+* 2 more repetitions of the last click.
 
-Long button and key presses cannot be extended, as the length of the click/press is timed using the release of the final button/key -- such shortcuts must be terminated with the raising of the final button/key.
+Your shortcut can include one or more modifiers (`Shift`, `Ctrl` and `Alt`). In this case, for a continuous shortcut, both the last key and the modifier must be pressed down when you execute the movement. For example, `Ctrl + E + E + pan` means that while moving the mouse left and/or right both  `E` and `Ctrl` must be pressed down.
 
 ---
 
-**Note:** You may need to switch off the "disable touchpad while typing" setting if you want to use extended shortcuts with a laptop touchpad.
+**Note:**
+If you are a MacOs user, your shortcuts will use `Cmd` instead of `Ctrl` and `Option` instead of `Alt`.
 
 ---
 
-# actions
+If you are defining a continuous shortcut, then the movement part of the shortcut must be executed while the last key, mouse or controller button is held down.
 
-Shortcuts are used to initiate _actions_ within darktable.
+For example, the shortcut `E + E + pan`, can be activated by pressing `E` twice, holding down `E` on the second press and moving the mouse horizontally while the key is pressed.
 
-An _action_ is usually (but not always) an operation that you might undertake using darktable's point-and-click user interface. For example:
+**Short and long key presses.** 
+By default, all key presses in a shortcut are _short_, i.e., the key is pressed and immediately released. However, the last repetition of a key in a shortcut can also be a _long_ key press, defined as holding down the key for a bit longer than the duration of a double click. Hence `E + E` and `E + E(long)` are two distinct shortcuts that can be assigned to different actions. The associated action triggers when the key is released, which entails that a shortcut ending with a long press cannot be used for a continuos shortcut.
 
--   Increase, decrease or reset sliders
--   Scroll through dropdown lists
--   Enable, expand or focus modules
--   Click buttons
--   Switch between views
+**Triggering multiple shortcuts at once.** 
+If both `E + mouse-scroll` and `F + mouse-scroll` are mapped to a slider (or if you have fallbacks enabled), then scrolling the mouse wheel while holding down both `E` and `F` will move both sliders.
```

Comment:
It would be nice to refer to the short cuts the same way as the preferences panel does, e.g. it doesn't call it "mouse scroll" just "scroll".